.github/workflows/pytest.yml

```diff
@@ -190,6 +186,177 @@ jobs:
           name: Unit Test Results (Python ${{ matrix.python-version }} ${{ matrix.test-markers }})
           path: pytest.xml
 
+  integration-tests-a:
+    name: Integration Tests (A)
```

Comment:
This makes sense since we're packaging all of the hyperopt tests together. However, group A also includes non-hyperopt tests. We could shift the groups around again, but we'll have to give the other groups generic names anyway.